src/extensions/monetize-block.js

```diff
@@ -170,12 +170,15 @@ const monetizeBlockControls = createHigherOrderComponent( ( BlockEdit ) => {
 
 		// Only show inspector options if set for block level monetization.
 		showInspector = false;
+		// console.log('meta', meta);
 		if ( typeof meta !== 'undefined' ) {
 			if ( typeof meta._coil_monetize_post_status === 'undefined' || ( typeof meta._coil_monetize_post_status !== 'undefined' && meta._coil_monetize_post_status === 'gate-tagged-blocks' ) ) {
 				showInspector = true;
 			}
 		}
 
+		// console.log('showInspector', showInspector);
```

Comment:
Does this debug need to stay in?

Reply:
Nope. Do I need to remove it to get approval?

Reply:
It's sloppy leaving commented out code behind, so yes please, remove it. :)

Reply:
Ok. Will do.